opensrp-reveal/src/main/assets/reveal-streets-satellite.json

```diff
@@ -201,6 +205,10 @@
             "Partially Received"
           ],
           "hsl(26, 84%, 56%)",
+          [
+            "Inactive"
+          ],
+          "hsl(0, 0%, 78%)",
           "hsl(0, 0%, 78%)"
```

Comment:
This is not required, since the task will be cancelled, also its the same as the default